source/PcAux/R/exportedHelperFunctions.R

```diff
@@ -297,7 +297,6 @@ calcTime <- function(pcAuxData, what) {
 
 ## Write machine status to a text files 
 writeStatus <- function(pcAuxData, outName, what) {
-    fileName <- file(outName)
-    writeLines(capture.output(pcAuxData$status[[what]]), fileName)
```

Comment:
You don't need the writeLines function. The capture.output function already writes the appropriate information to disk.